recipe/build.sh

```diff
@@ -3,6 +3,9 @@
 if [ "$(uname)" == "Darwin" ]; then
     export CXX="${CXX} -stdlib=libc++"
     export LDFLAGS="${LDFLAGS} -Wl,-rpath,$PREFIX/lib"
+else
+    # Shared Fortran libs on OSX unsupported as of 1.8.18
+    FORTRAN_FLAGS=("--enable-fortran" "--enable-fortran2003")
```

Comment:
I don't have a Mac, so I've been testing through Travis here - my impression is that Apple ships those with OSX. We could try to delete the shared libraries that are built, but we'd need to delete between ```make``` and ```make check```.

Given the fortran test suite is totally failing on OSX, I'd be skeptical of whether it's safe to rely on.